.github/workflows/unit-tests.yml

```diff
@@ -70,41 +70,19 @@ jobs:
       #   no additional versions are included in the array.
       # - experimental: Whether the build is "allowed to fail".
       matrix:
```

Comment:
What with PHPCSUtils now being added as a dependency, should it be added to the matrix ? Or maybe the matrix/workflow should be changed to make use of the Composer `--prefer-lowest`/`--prefer-highest` options to simplify the maintenance of the version combinations ?

(yes, something I'm looking to do for WPCS as well)

Reply:
https://github.com/Automattic/VIP-Coding-Standards/issues/743 opened to handle this separately.